README.md

```diff
@@ -8,5 +8,31 @@
 A python package contains functions that facilitate working with various
 versions of Ansible 2.14 and newer.
 
+## Documentation
+
 Documentation is available at
 [ansible-compat.readthedocs.io](https://ansible-compat.readthedocs.io/).
+
+## Communication
+
+Join the Ansible forum to ask questions, get help, and interact with the
+community.
+
+- [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
+  Please add appropriate tags if you start new discussions, for example the
+  `compat` tag.
```

Comment:
```suggestion
  `compat` tag.
```
There's not the `compat` tag in the forum as far as i can see. I'm not sure if it's needed. I would remove it for now and let the maintainers request it and update the line later.